oci/kratos/image.yaml

```diff
@@ -0,0 +1,12 @@
+version: 1
+upload:
+  - source: "canonical/kratos-rock"
+    commit: 396bf3a71cb65f97cf853540117858d6859ef43b
+    directory: .
+    release:
+      1.1.0-22.04:
+        risks:
+          - stable
+          - candidate
+          - edge
+        end-of-life: "2026-01-01T00:00:00Z"
```

Comment:
This is a very long EOL (1+y) for something that is not in our archives. If this 1.1.0 version is dropped from upstream tomorrow, will you and your team keep maintaining it? 

We tend to be much more conservative for things that are not in the archives.